ci/azure/linux.yml

```diff
@@ -28,6 +28,13 @@ jobs:
         PYTHON_VERSION: $(PYTHON_MAJOR_VERSION).$(PYTHON_MINOR_VERSION)
         PYTHON_NO_DOT_VERSION: $(PYTHON_MAJOR_VERSION)$(PYTHON_MINOR_VERSION)
         PYTEST_MARK_EXPRESSION: "not udf"
+      py38:
```

Comment:
can you move the duplicated variables in the matrix to the variables tab (I think this is where they go)